shopify/product-reviews-template/api/actions/sendReviewRequests.js

```diff
@@ -0,0 +1,47 @@
+import { SendReviewRequestsGlobalActionContext } from "gadget-server";
+import { v4 as uuidv4 } from "uuid";
+
+/**
+ * @param { SendReviewRequestsGlobalActionContext } context
+ */
+export async function run({ params, logger, api, connections }) {
```

Comment:
Where is this action called from?
Is it supposed to use a scheduler trigger?

Reply:
Yep, forgot about it and added it in